fairscale/nn/misc/checkpoint_activations.py

```diff
@@ -113,23 +125,21 @@ def forward(  # type: ignore
             unpacked_args, unpacked_kwargs = unpack_kwargs(kwarg_keys, args)
             outputs = run_function(*unpacked_args, **unpacked_kwargs)
 
-        if isinstance(outputs, torch.Tensor):
-            return outputs
-        else:
+        if not isinstance(outputs, torch.Tensor):
```

Comment:
nice! I love early return statements in the code which avoid if/else constructs.

Reply:
yeah, early return is often a source of bug. :-)